### PR TITLE
fix(sidenav): make `focus-trap` occupy full height of sidenav

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

File: src/lib/sidenav/sidenav.scss [hunk 1]
```diff
@@ -108,8 +108,15 @@ md-sidenav {
   z-index: 3;
   min-width: 5%;
 
-  // TODO(kara): revisit scrolling behavior for sidenavs
-  overflow-y: auto;
+  > focus-trap {
```
Comment (jelbourn) [inline src/lib/sidenav/sidenav.scss]: `.md-sidenav-focus-trap` etc.

**Comment by mmalerba** [comment]
comments addressed

File: src/lib/core/a11y/focus-trap.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <div *ngIf="!disabled" tabindex="0" (focus)="focusLastTabbableElement()"></div>
-<div #trappedContent><ng-content></ng-content></div>
+<div #trappedContent class="focus-trap-content"><ng-content></ng-content></div>
```
Comment (jelbourn) [inline src/lib/core/a11y/focus-trap.html]: Also prefix on `.focus-trap-content`. This is as good a time as any to introduce the `cdk-` prefix.

Comment (mmalerba) [inline src/lib/core/a11y/focus-trap.html]: shouldn't we wait and do that all at once?

Comment (jelbourn) [inline src/lib/core/a11y/focus-trap.html]: I don't see any reason not to start now

**Pull request merged**